nat-lab/tests/test_mesh_exit_through_peer.py

```diff
@@ -111,3 +111,17 @@ async def test_mesh_exit_through_peer(
         assert ip_alpha == ip_beta
         assert alpha_conn_tracker.get_out_of_limits() is None
         assert beta_conn_tracker.get_out_of_limits() is None
+
+        # Testing if the exit node is cleared after disabling meshnet. See LLT-4266 for more details.
+        await client_alpha.set_mesh_off()
+        await testing.wait_long(
+            client_alpha.wait_for_event_peer(
+                beta.public_key, [State.Disconnected], list(telio.PathType)
+            )
+        )
```

Comment:
Will this fail if there is a `connecting` state before the `disconnected` state`? Please correct me if I'm wrong but I don't think it does, and I think the next block that expects a `TimeoutError` will not verify either that there's no `connecting event` before `disconnecting` since this first block will clear those events

Reply:
Fixed

Reply:
Fixed.